### PR TITLE
refactor: unify desktop content width to 940px across all pages

### PR DESCRIPTION
**Pull request by llun**
## Summary

Aligns the app with the design system handoff (`ui_kits/web/index.html`), where **every desktop navigation tab uses one unified content width — 940px**. Previously the app had two tiers — `max-w-2xl` (timeline, search, bookmarks, notifications, profile, status) and `max-w-4xl` (settings, fitness, admin) — plus Messages opting into a full-width override, so the content column jumped around as you switched tabs.

Now a single `--container-content: 940px` token (exposed as the `max-w-content` Tailwind utility) drives both the `(timeline)` layout wrapper and the `PageHeader` title row. Section layouts and Messages inherit it instead of declaring their own width. Mobile/tablet are unaffected.

## Changes

- **`globals.css`** — add the `--container-content` token; remove the `data-layout-width="wide"` `:has` override rule.
- **`(timeline)/layout.tsx`** — wrapper `max-w-2xl` → `max-w-content`.
- **`page-header.tsx`** — drop the now-vestigial `contentWidth` prop; always center the title row at `max-w-content`.
- **`settings` / `fitness` / `admin` layouts** — drop `contentWidth="wide"` and the inner `max-w-4xl` / `data-layout-width` cap; the wrapper is a plain `w-full` div.
- **`MessagesPage.tsx`** — drop `data-layout-width="wide"`; inherits the unified width.
- **Test + docs** — update the Messages width assertion and the AGENTS.md / CLAUDE.md layout guidance.

## Verification

Ran locally (SQLite + mock user) at a 1440px desktop viewport and measured every route — all now report **`width: 940px`, `left: 390`** (centered beside the 280px sidebar):

| Route | Before | After |
|---|---|---|
| timeline / search / bookmarks / notifications | 672 (`max-w-2xl`) | **940** |
| profile / status | 672 | **940** |
| settings / fitness / admin | 896 (`max-w-4xl`) | **940** |
| messages | full-width | **940** (comfortable two-pane, not cramped) |

- `yarn run prettier --write` ✓
- `yarn lint` — 0 errors ✓
- `yarn build` ✓
- `yarn test` — 3299 passed ✓

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -48,6 +48,12 @@
   --radius-2xl: calc(var(--radius) + 8px);
   --radius-3xl: calc(var(--radius) + 12px);
   --radius-4xl: calc(var(--radius) + 16px);
+
+  /* Unified desktop content width. Every top-level page in the (timeline)
+     route group centers its content (and its page-header title row) at this
+     single width, so the column stays aligned as you switch tabs. Exposed as
+     the `max-w-content` utility via Tailwind's `--container-*` namespace. */
+  --container-content: 940px;
```
[the remark this answers is not in the document]
Comment (llun) [reply app/globals.css:56]: Thanks for flagging — I checked this carefully and the opposite is true: in **Tailwind CSS v4 the `max-w-*` utilities are generated from the `--container-*` namespace**, not `--width-*`. This is the same mechanism that makes the built-in `max-w-2xl` resolve to `--container-2xl`. So `--container-content: 940px` correctly produces the `max-w-content` utility.

Concrete evidence from this branch:

- **Production build CSS** (`.next/static/chunks/*.css`) contains the literal rule:
  ```css
  .max-w-content{max-width:940px}
  ```
- **In-browser**, `.timeline-content` (which carries `max-w-content`) computes to `max-width: 940px` on every desktop page.

`--width-content` is **not** a recognized Tailwind v4 theme namespace, so switching to it would remove the `max-w-content` utility and drop the 940px constraint entirely — the opposite of the intended fix. Keeping `--container-content`. Resolving as not-an-issue.

File: AGENTS.md [hunk 2]
```diff
@@ -132,17 +132,18 @@
...
+- Use `PageHeader` from `@/lib/components/page-header` for every page title in the `(timeline)` route group. By default it renders the sticky, full-width chrome (translucent background + backdrop blur + bottom border) and centers the title above the content column. Pages always call `<PageHeader title="…" description="…" actions={…} />`; they don't need to know which sub-nav pattern (if any) wraps them.
+- **Unified desktop content width.** Every top-level page in the `(timeline)` group shares **one** content width on desktop so the column stays aligned as you switch tabs. The `(timeline)` layout wrapper centers content at `max-w-content` (a single `--container-content: 940px` token defined in `app/globals.css`'s `@theme`), and `PageHeader` centers its title row at the same `max-w-content`. There is **no** per-page width tier any more: do **not** reintroduce the old two-tier `max-w-2xl` (timeline) / `max-w-4xl` (sections) split, a `contentWidth` prop on `PageHeader`, or the `data-layout-width="wide"` opt-in CSS rule. Section layouts (settings, fitness, admin) and Messages all inherit the unified `max-w-content` from the wrapper — they don't set their own width.
```
[the remark this answers is not in the document]
Comment (llun) [reply AGENTS.md:137]: The underlying premise (that `--container-content` doesn't generate `max-w-content`) is incorrect — see the reply on `app/globals.css`. The production build emits `.max-w-content{max-width:940px}` and the rendered pages compute `max-width: 940px`, confirming `--container-*` is the correct Tailwind v4 namespace for `max-w-*`. The docs intentionally reference `--container-content`, so no change is needed here.

File: CLAUDE.md [hunk 3]
```diff
@@ -17,7 +17,8 @@
...
+- Every top-level page in the `(timeline)` group shares **one** desktop content width: the layout wrapper and `PageHeader` both center at `max-w-content` (a single `--container-content: 940px` token in `app/globals.css`). Do **not** reintroduce the old two-tier `max-w-2xl`/`max-w-4xl` split, a `contentWidth` prop, or the `data-layout-width="wide"` opt-in — sections and Messages inherit the unified width.
```
[the remark this answers is not in the document]
Comment (llun) [reply CLAUDE.md:21]: The underlying premise (that `--container-content` doesn't generate `max-w-content`) is incorrect — see the reply on `app/globals.css`. The production build emits `.max-w-content{max-width:940px}` and the rendered pages compute `max-width: 940px`, confirming `--container-*` is the correct Tailwind v4 namespace for `max-w-*`. The docs intentionally reference `--container-content`, so no change is needed here.

**Comment by llun** [comment]
### Subagent review pass (round 1)

Dispatched two independent review subagents over `d4d102e89..HEAD`:

1. **Correctness / conventions reviewer** — checked for dead references to the removed two-tier system (`data-layout-width`, `contentWidth` prop, `:has` rule), stray `max-w-2xl`/`max-w-4xl` content columns, logic regressions in the edited TSX, and AGENTS.md convention violations.
2. **CSS / Tailwind-v4 / responsive reviewer** — checked PageHeader breakout alignment across breakpoints (mobile / 72px rail / 280px sidebar), tablet overflow, Messages internal grid at the new 940px cap, and leftover CSS.

**Outcome:** no high-confidence defects. Both confirmed the unified `max-w-content` token is applied consistently and behaves correctly at every breakpoint.

**One leftover addressed:** the `.timeline-content` class on the `(timeline)` layout wrapper was now inert — its only consumer (`.timeline-content:has([data-layout-width='wide'])`) was deleted with the old system, so the class matched no CSS/JS. Removed it in `refactor: drop the now-inert timeline-content class` to avoid implying a rule still targets it. Verified after the change: production build still emits `.max-w-content{max-width:940px}`, lint clean, MessagesPage tests pass.

File: app/globals.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (llun) [reply app/globals.css:56]: I ran this as an actual experiment on this branch to settle it definitively, and `--container-content` is correct — switching to `--width-content` would **break** the feature. Clean `yarn build` results:

| Token | Generated CSS contains `.max-w-content`? |
|---|---|
| `--container-content: 940px` (current) | ✅ `.max-w-content{max-width:940px}` |
| `--width-content: 940px` (suggested) | ❌ no `.max-w-content` rule, no `940px` at all |

In Tailwind CSS v4 the `--container-*` namespace is exactly what powers the `max-w-*` utilities — it's the same scale the built-in `max-w-2xl` reads from (`--container-2xl`), and it drives both `max-w-*` and the `@container` query variants. There is no `--width-*` theme namespace, so `--width-content` compiles to nothing and the column would lose its 940px cap.

I've verified the current code three ways: the production-build CSS literally emits `.max-w-content{max-width:940px}`, the rendered pages compute `max-width: 940px`, and the `--width-content` variant produces no utility at all. Keeping `--container-content`. Resolving as not-an-issue.

File: AGENTS.md [hunk 4]
```diff
@@ -133,17 +133,18 @@
...
```
[the remark this answers is not in the document]
Comment (llun) [reply AGENTS.md:137]: The premise here is incorrect — see the reply on `app/globals.css`. I empirically tested both tokens with a clean `yarn build`: `--container-content` emits `.max-w-content{max-width:940px}`, while the suggested `--width-content` emits **no** `max-w-content` utility at all (there is no `--width-*` namespace in Tailwind v4). So the docs correctly reference `--container-content`; changing them to `--width-content` would document a token that doesn't work. No change needed.

File: CLAUDE.md [hunk 5]
```diff
@@ -18,7 +18,8 @@
...
```
[the remark this answers is not in the document]
Comment (llun) [reply CLAUDE.md:21]: The premise here is incorrect — see the reply on `app/globals.css`. I empirically tested both tokens with a clean `yarn build`: `--container-content` emits `.max-w-content{max-width:940px}`, while the suggested `--width-content` emits **no** `max-w-content` utility at all (there is no `--width-*` namespace in Tailwind v4). So the docs correctly reference `--container-content`; changing them to `--width-content` would document a token that doesn't work. No change needed.

**Comment by llun** [comment]
### Subagent review pass (round 2 — adversarial)

Ran a fresh skeptical/adversarial review subagent (lens: test coverage, prop-removal fallout, accessibility, edge cases, breakout math). It re-verified the cleanup is complete (no stray `contentWidth`, `data-layout-width`, `timeline-content`, or `max-w-2xl/4xl` content columns; `(nosidebar)` group unaffected; breakout math correct at every breakpoint) and surfaced **one genuine gap**:

- **No regression test guarded the unified width.** The dropped `data-layout-width` assertion in `MessagesPage.test.tsx` was correct to remove, but nothing replaced it — a future edit reverting a page to `max-w-2xl`/`max-w-4xl`, or diverging the `PageHeader` title row from the content column, would leave the suite green.

**Fix:** added `lib/components/page-header.test.tsx` (`test: guard the unified max-w-content header width`) asserting the sticky header centers its title row at `max-w-content`, and that section mode imposes no width of its own. `PageHeader` renders on every `(timeline)` page, so it's the highest-value guard. Tests pass; lint/build clean.

File: app/globals.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (llun) [reply app/globals.css:56]: Same suggestion as the prior two rounds — already settled empirically and skipping per verification. A clean `yarn build` on this branch with `--width-content: 940px` produces **no** `.max-w-content` rule (the utility isn't generated), while `--container-content: 940px` emits `.max-w-content{max-width:940px}`. In Tailwind v4 `--container-*` is the namespace that powers `max-w-*` (same scale the built-in `max-w-2xl` reads from); there is no `--width-*` namespace. Applying this change would remove the 940px cap entirely, so it's intentionally not applied. Resolving.
